designs/BlackParrot/descriptor.yaml

```diff
@@ -0,0 +1,452 @@
+    - src/basejump_stl/bsg_mux2_gatestack.sv
+    - src/basejump_stl/bsg_tag_trace_replay.sv
+    - src/basejump_stl/bsg_tag_master.sv
+    - src/basejump_stl/bsg_tag_client.sv
+    - src/basejump_stl/bsg_tag_client_unsync.sv
+    - src/basejump_stl/bsg_nonsynth_axi_mem.sv
+    - src/basejump_stl/bsg_nonsynth_test_rom.sv
+    - src/bp/bp_me_nonsynth_pkg.sv
+    - src/bp/bp_nonsynth_cfg_loader.sv
+    - src/bp/bp_nonsynth_watchdog.sv
+    - src/bp/bp_nonsynth_host.sv
+    - src/bp/bp_nonsynth_if_verif.sv
+    - src/bp/bp_nonsynth_nbf_loader.sv
+    - src/bp/bp_nonsynth_perf.sv
+    - src/bp/bp_monitor.cpp
+    - src/bp/bp_nonsynth_dram.sv
+    - src/bp/testbench.sv
+    - src/basejump_stl/bsg_nonsynth_reset_gen.sv
+    - src/basejump_stl/bsg_nonsynth_clock_gen.sv
+
+  verilogIncludeFiles:
+    - src/bp/bp_be_ctl_pkgdef.svh
+    - src/bp/bp_be_dcache_defines.svh
+    - src/bp/bp_be_dcache_pkgdef.svh
+    - src/bp/bp_be_defines.svh
+    - src/basejump_stl/bsg_defines.sv
+    - src/basejump_stl/bsg_extra_defines.svh
+    - src/basejump_stl/bsg_noc_links.svh
+    - src/basejump_stl/bsg_wormhole_router.svh
+    - src/bp/bp_common_accelerator_pkgdef.svh
+    - src/bp/bp_common_addr_defines.svh
+    - src/bp/bp_common_addr_pkgdef.svh
+    - src/bp/bp_common_aviary_cfg_pkgdef.svh
+    - src/bp/bp_common_aviary_custom_defines.svh
+    - src/bp/bp_common_aviary_defines.svh
+    - src/bp/bp_common_aviary_pkgdef.svh
+    - src/bp/bp_common_bedrock_if.svh
+    - src/bp/bp_common_bedrock_pkgdef.svh
+    - src/bp/bp_common_bedrock_wormhole_defines.svh
+    - src/bp/bp_common_cache_engine_if.svh
+    - src/bp/bp_common_cache_engine_pkgdef.svh
+    - src/bp/bp_common_cache_pkgdef.svh
+    - src/bp/bp_common_cfg_bus_defines.svh
+    - src/bp/bp_common_cfg_bus_pkgdef.svh
+    - src/bp/bp_common_clint_pkgdef.svh
+    - src/bp/bp_common_core_if.svh
+    - src/bp/bp_common_core_pkgdef.svh
+    - src/bp/bp_common_defines.svh
+    - src/bp/bp_common_host_pkgdef.svh
+    - src/bp/bp_common_rv64_csr_defines.svh
+    - src/bp/bp_common_rv64_instr_defines.svh
+    - src/bp/bp_common_rv64_pkgdef.svh
+    - src/bp/bp_common_test_defines.svh
+    - src/bp/bp_fe_pc_gen_pkgdef.svh
+    - src/bp/bp_fe_icache_defines.svh
+    - src/bp/bp_fe_icache_pkgdef.svh
+    - src/bp/bp_fe_defines.svh
+    - src/bp/bp_me_cce_defines.svh
+    - src/bp/bp_me_cce_inst_defines.svh
+    - src/bp/bp_me_cce_inst_pkgdef.svh
+    - src/bp/bp_me_cce_pkgdef.svh
+    - src/bp/bp_me_defines.svh
+    - src/bp/bp_top_defines.svh
+    - src/hardfloat/HardFloat_consts.vi
+    - src/hardfloat/HardFloat_specialize.vi
+    - src/hardfloat/HardFloat_localFuncs.vi
+    - src/basejump_stl/bsg_cache.svh
+    - src/basejump_stl/bsg_tag.svh
+  verilogDefines:
+    den2048Mb: 1
+    sg5: 1
+    x16: 1
+    FULL_MEM: 1
+  cppSourceFiles:
+    - src/basejump_stl/bsg_mem_dma.cpp
+  cppIncludeFiles:
+    - src/basejump_stl/bsg_mem_dma.hpp
+  topModule: testbench
+  mainClock: testbench.dut_clk
+  verilatorArgs:
```

Comment:
Please add `--autoflush` to `verialtorArgs` so lines are printed when ready. This makes progress easier to see.